### ISSUE TITLE
startPrivateConversationWithPersonId with Spark does not set conversation context

### ISSUE THREAD

**Issue by joebruzek** [howdyai/botkit#864]
When I use `startPrivateConversationWithPersonId(personid, function(err, convo){})` the resulting conversation object has no user or channel set in its context and as a result does not work properly, since new messages with that user will not match the conversation that the bot started.

Here is the `conversation.context` I get when creating a conversation with `startPrivateConversationWithPersonId()`:
```
context: {
  user: undefined,
  channel: undefined,
  bot: { ... }
}
```

**Comment by juxe** [comment]
I'm having this issue as well. The following example is not working:

**Comment by benbrown** [comment]
This will be addressed by #942

**Comment by benbrown** [comment]
This has been fixed in 0.5.6
